javascript/core/flow-control/flow-control-discussion-insight.md

```diff
@@ -0,0 +1,62 @@
+---
+author: kapnobatai136
+
+tags:
+  - discussion
+
+type: normal
+
+category: discussion
+
+links:
+  - >-
+    [Example](https://repl.it/@enkicontent/JavaScriptFlowControl){website}
+
+---
+
+# Test Yourself
+
+---
+
+## Content
+
+Now that you learned some JavaScript, it's time to apply it!
+
+Let's say you are a farmer, and you need someone to count all the legs of your animals.
+
+You remember that you learned JavaScript with Enki and that you can do this through coding.
+
+How would you write a function that takes as input the number of each animal and returns the total number of legs?
```

Comment:
```suggestion
How would you write a function that takes as input the number of all animals and returns the total number of legs?
```

Would this maybe sound better?

That "number of each animal" sounds a bit weird imo. I had to re-read it a few times to understand what is meant

Reply:
Based on the attached repl, the text should also mention the animal name.

How about something like this?

````suggestion
How would you write a function that takes two inputs, the type of an animal and how many of those animals are at the farm, and returns the total number of legs?

For example, in our Enki farm, we count the number of legs for our cows like:

```js
const animal = 'cow';
const numberOfAnimals = 10;

countLegs(animal, numberOfAnimals); // 40
```
````

Reply:
Can also put the example somewhere around line 37, or in the footnote.